README.md

```diff
@@ -126,6 +127,13 @@ make
 sudo make install
 ```
 
+### Build options ###
+By passing `NOTIFICATIONS=1` to the `make` call, notifications via
+libnotify are enabled. Necessary libraries are 
+`gmodule-2.0`, `glib-2.0`, and `notify`. If these libraries are
+named differently on the build system, the make variable
```

Comment:
Do you know how different this 'may' be?

Is there a way to have an autoconf or something to configure the correct additional libraries if on a certain OS type?

Reply:
Debian, Ubuntu, CentOS, Fedora, SuSE all have these libraries, so I really guess that is fine. The default glib is version 3, but libnotify requires stuff from glib2.

So in most cases I think this is fine. Do you have specific distribution you want me to check?

Reply:
I forgot to mention, yes, there is different and better way to get the correct libs: `pkg-config --libs libnotify`. And with the autoconf-ication I am working on libs will be automatically searched with `pkg-config` anyway, but for now I did refrain from using it in the `Makefile`, also because it wasn't used by now, but could in principle be used for `sqlite3` and `curl`, too.

Reply:
> So in most cases I think this is fine. Do you have specific distribution you want me to check?

No - this was more in general so that if users got stuck, they could look to the readme for hints as to the packages needed.

> And with the autoconf-ication I am working on libs will be automatically searched with pkg-config anyway, but for now I did refrain from using it in the Makefile, also because it wasn't used by now, but could in principle be used for sqlite3 and curl, too.


I think if we are 'switching' to have the possibility of having more library dependencies, switching 'early' to a better method to ensure the dependencies are there is a good move.